pkgs/development/libraries/libtelnet/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, pkgconfig, autoreconfHook, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "libtelnet-${version}";
+  version = "0.21";
+
+  src = fetchFromGitHub {
+    owner = "seanmiddleditch";
+    repo = "libtelnet";
+    rev = "45f2d5cfcf383312280e61c85b107285fed260cf";
```

Comment:
The tagged 0.21 does not compile. The latest commit does. should we just put `0.22-git45f2d5`?